DOC-3132: New `revisionhistory_allow_restore` option to control restoration of old revisions. (#3615)
tinymce/tinymce-docs: modules/ROOT/pages/7.7.0-release-notes.adoc

For information on the **Image Optimizer** plugin, see: xref:uploadcare.adoc[Image Optimizer].

=== Revision History

The {productname} {release-version} release includes an accompanying release of the **Revision History** premium plugin.

**Revision History** includes the following addition.

==== New `revisionhistory_allow_restore` option to control restoration of old revisions  
// #TINY-11746

> 

Author: ```suggestion
```

For information on the **Image Optimizer** plugin, see: xref:uploadcare.adoc[Image Optimizer].

=== Revision History

The {productname} {release-version} release includes an accompanying release of the **Revision History** premium plugin.

**Revision History** includes the following addition.

==== New `revisionhistory_allow_restore` option to control restoration of old revisions  
// #TINY-11746

> Previously, integrators had no way to control whether users could restore old revisions in the **Revision History** plugin.

Reviewer: ```suggestion
```

> {prductname} {release-version} introduces a new `revisionhistory_allow_restore` option, which provides integrators with a way to manage this behavior. By default, it is set to `+true+`, setting it to `+false+` prevents users from restoring previous versions.

Reviewer 2: Typo. Should be {productname}

Reviewer 3: ```suggestion
{productname} {release-version} introduces a new `revisionhistory_allow_restore` option, which provides integrators with a way to manage this behavior. By default, it is set to `+true+`, setting it to `+false+` prevents users from restoring previous versions.
```

> {productname} {release-version} introduces a new `revisionhistory_allow_restore` option, which provides integrators with a way to manage this behavior. By default, it is set to `+true+`, setting it to `+false+` prevents users from restoring previous versions.

Reviewer: ```suggestion
{productname} {release-version} introduces a new `revisionhistory_allow_restore` option, which provides a way to control if a user can restore revisions. By default, it is set to `+true+`, setting it to `+false+` prevents users from restoring previous versions.
```

This improvement enhances access control within the **Revision History** plugin, providing greater flexibility for managing revision restoration.

> For more information, see: xref:revisionhistory.adoc[Revision History].

Reviewer 3: ```suggestion
For more information on the **Revision History** plugin, see: xref:revisionhistory.adoc[Revision History].
```

Reviewer 3: @kemister85 Maybe we should follow the regular format for the more information section

For information on the **Image Optimizer** plugin, see: xref:uploadcare.adoc[Image Optimizer].

=== Revision History

The {productname} {release-version} release includes an accompanying release of the **Revision History** premium plugin.

**Revision History** includes the following addition.

==== New `revisionhistory_allow_restore` option to control restoration of old revisions  
// #TINY-11746

Previously, integrators had no way to control whether users could restore old revisions in the **Revision History** plugin.

{prductname} {release-version} introduces a new `revisionhistory_allow_restore` option, which provides integrators with a way to manage this behavior. By default, it is set to `+true+`, setting it to `+false+` prevents users from restoring previous versions.

This improvement enhances access control within the **Revision History** plugin, providing greater flexibility for managing revision restoration.

For more information, see: xref:revisionhistory.adoc[Revision History].

> ==== New `revisionhistory_allow_restore` option to control if users can restore old versions.

Author: ```suggestion
```
